ArchiSteamFarm/Bot.cs

```diff
@@ -1173,8 +1173,19 @@ public sealed class Bot : IDisposable {
 			// We must escape our message prior to sending it
 			message = Escape(message);
 
-			for (int i = 0; i < message.Length; i += maxMessageLength) {
-				string messagePart = message.Substring(i, Math.Min(maxMessageLength, message.Length - i));
+			int i = 0;
+
+			while (i < message.Length) {
+				int partLength;
+
+				if (message.Length - i > maxMessageLength) {
+					int lastNewLine = message.LastIndexOf(Environment.NewLine, i + maxMessageLength, maxMessageLength, StringComparison.Ordinal);
+					partLength = lastNewLine > i ? lastNewLine - i : maxMessageLength;
+				} else {
+					partLength = message.Length - i;
+				}
+
+				string messagePart = message.Substring(i, partLength);
```

Comment:
No, it looks good like this, `…` is added before the newline, so in steam its `…` on separate line and than actual data. Changing this will make output ugly.